docs/projects/ad7606x_fmcz/index.rst

```diff
@@ -0,0 +1,450 @@
+   -   **JP5** - B - Parallel interface
+   -   **JP7, JP12, JP13, JP14** - B - Differential operation
+
+Resources
+-------------------------------------------------------------------------------
+
+Systems related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :dokuwiki:`AD7606X HDL Reference Design <resources/eval/user-guides/ad7606x-fmc/hdl>`
+-  :dokuwiki:`AXI_AD7616 <resources/fpga/docs/axi_ad7606x>`
+-  :dokuwiki:`AD7606-No-OS Driver <resources/tools-software/uc-drivers/ad7606>`
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Product datasheet: :adi:`AD7606`
+
+-  `UG-1870, Evaluation Board User Guide <https://www.analog.com/media/en/technical-documentation/user-guides/eval-ad7606c-fmcz-ug-1870.pdf>`__
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`ad7616x_fmc HDL project <main:projects/ad7606x_fmc>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - AD_EDGE_DETECT
+     - :git-hdl:`library/common/ad_edge_detect.v <main:library/common/ad_edge_detect.v>`
+     -
+   * - AXI_AD7606
+     - :git-hdl:`library/axi_ad7606x <main:library/axi_ad7606x>` *
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_ad7606x>`
+   * - AXI_CLKGEN
+     - :git-hdl:`library/axi_dmac <main:library/axi_clkgen>` **
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_clkgen>`
+   * - AXI_DMAC
+     - :git-hdl:`library/axi_dmac <master:library/axi_dmac>`
+     - :ref:`here <axi_dmac>`
+   * - AXI_HDMI_TX
+     - :git-hdl:`library/axi_hdmi_tx <main:library/axi_hdmi_tx>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_hdmi_tx>`
+   * - AXI_I2S_ADI
+     - :git-hdl:`library/axi_i2s_adi <main:library/axi_i2s_adi>`
+     -
+   * - AXI_PWM_GEN
+     - :git-hdl:`library/axi_pwm_gen <main:library/axi_pwm_gen>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_pwm_gen>`
+   * - AXI_SPDIF_TX
+     - :git-hdl:`library/axi_spdif_tx <main:library/axi_spdif_tx>`
+     -
+   * - AXI_SYSID
+     - :git-hdl:`library/axi_sysid <main:library/axi_sysid>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_sysid>`
+   * - AXI_SPI_ENGINE
+     - :git-hdl:`library/spi_engine/axi_spi_engine <main:library/spi_engine/axi_spi_engine>`  **
+     - :ref:`here <spi_engine axi>`
+   * - SPI_ENGINE_EXECUTION
+     - :git-hdl:`library/spi_engine/spi_engine_execution <main:library/spi_engine/spi_engine_execution>` **
+     - :ref:`here <spi_engine execution>`
+   * - SPI_ENGINE_INTERCONNECT
+     - :git-hdl:`library/spi_engine/spi_engine_interconnect <main:library/spi_engine/spi_engine_interconnect>` **
+     - :ref:`here <spi_engine interconnect>`
+   * - SPI_ENGINE_OFFLOAD
+     - :git-hdl:`library/spi_engine/spi_engine_offload <main:library/spi_engine/spi_engine_offload>` **
+     - :ref:`here <spi_engine offload>`
+   * - SYNC_BITS
```

Comment:
Is this IP used in the project? 
util_cpack2 should be present here, as it is used (if you look at the Makefile of this project)

Reply:
Solved, these will be added in the code in PR #1206

Reply:
Done.